packages/web/pages/dashboard/new-post.tsx

```diff
@@ -89,6 +89,8 @@ const NewPostPage: NextPage<NewPostPageProps> = ({ defaultImage }) => {
   const [saving, setSaving] = React.useState<boolean>(false)
   const [errorMessage, setErrorMessage] = useState('')
 
+  // TODO URGENT: FIGURE OUT CHANGES IN THIS FILE
```

Comment:
This is concerning 😅 

Hmm, I'm not sure if this is going to correctly update the cache, I'm assuming the `limit` and `skip` keys are part of the cache key, so this only updates the cache for the query getting the first post by the current user (which we don't do anywhere). Maybe using [refetchQueries](https://www.apollographql.com/docs/react/data/mutations/#refetching-queries) instead of `update` would make more sense?

Reply:
Ah yes lol I put this here to be eye-catching 😂 Just sent ya a DM